ListApp/Sources/ListApp.Api.Services/Bootstrapper/ServicesApiBootstrapper.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace ListApp.Api.Services.RouteHelper
```

Comment:
Now the namespace does not correspond the folder structure. There's a ReSharper refactoring to fix that.